project/Dependencies.scala

```diff
@@ -11,23 +11,23 @@ object Dependencies {
     val crossScala = Seq("2.11.5", "2.10.4")
     val scalaVersion = crossScala.head
     val scalaStmVersion  = sys.props.get("akka.build.scalaStmVersion").getOrElse("0.7")
-    val scalaTestVersion = sys.props.get("akka.build.scalaTestVersion").getOrElse("2.1.3")
-    val scalaCheckVersion = sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.11.3")
+    val scalaTestVersion = sys.props.get("akka.build.scalaTestVersion").getOrElse("2.1.7")
```

Comment:
We'll even hop on `2.2.4` with just one minor change (I have that in my PR) :-)
// Feel free to merge this one first.